docs/eventing/sources/README.md

```diff
@@ -15,8 +15,11 @@ a class of events from a particular software system. Since different event sourc
 by different Custom Resources, this page provides an index of the available source resource types as
 well as links to installation instructions.
 
-This is a non-exhaustive list of Event sources for Knative.
+For details how to write an Event Source consult one of the following tutorials:
+* [Write an EventSource using ContainerSource](../write-event-source-easy-way)
+* [Write an EventSource Controller](../write-event-source)
```

Comment:
Use "Event Source" throughout as per title, not "EventSource". Also I don't think there's any need to capitalize it generally